src/helpers/edit2var.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+module.exports = (editUrl, type) => {
```

Comment:
I guess you take the implementation from fedora.
I suggest that
- we add a comment to provide the link of the original source (as a reminder or if we need to improve the function in the future
- we remove the type parameter which is useless here

Reply:
done in a41c8513